lib/grape-swagger-rails.rb

```diff
@@ -3,10 +3,16 @@
 module GrapeSwaggerRails
   class Options < OpenStruct
     def before_filter(&block)
+      ActiveSupport::Deprecation.warn('this option is deprecated and going to be removed soon. ' \
```

Comment:
Should probably start with a capital `T` to be consistent and have a period after instead?